### ISSUE TITLE
Use webp for thumbnails

### ISSUE THREAD

**Issue by leiserfg** [photoview/photoview#406]
JPEG is an old fashion image format and fortunately, all major browsers (including Safari 14) can handle WebP so I will love to be able to use take advantage of its bigger compression ratio. If not as default (as there are users with old safaris), at least under a config flag. That should make the cache sizes decrease.

**Comment by viktorstrate** [comment]
I agree, this could potentially save a lot of storage and increase performance + data usage.
I do think it's necessary to add a flag to allow users to use JPEG as it looks like still many Apple devices doesn't support it.
